test/Npgsql.Specification.Tests/Utility.cs

```diff
@@ -7,7 +7,7 @@ public static class Utility
     {
         public static void ExecuteNonQuery(IDbFactoryFixture factoryFixture, string sql)
         {
-            using (var connection = factoryFixture.Factory.CreateConnection())
+            using (var connection = factoryFixture.Factory.CreateConnection()!)
```

Comment:
Probably not your fault and not a Npgsql Problem but a Factory that can create a `null` Connection is a sad API.

Reply:
Preview 8 will definitely add another level of breakage to our already-broken CI-publish-step since it [requires Visual Studio v16.8](https://dotnet.microsoft.com/download/dotnet/5.0) which is [not in sight for GitHub Actions](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#visual-studio-enterprise-2019).

Reply:
:rofl: 

I tried to fight for changing precisely this, and it didn't work :) If you're interested, starting reading from here: https://github.com/dotnet/runtime/pull/689#pullrequestreview-437632880

But yeah, at the end of the day it's a sad API.